Dockerfile

```diff
@@ -23,4 +23,5 @@ RUN wget -q -O ${GLIBC_KEY_FILE} ${GLIBC_KEY} \
 COPY --from=build-environment /opt/foundry/target/release/forge /usr/local/bin/forge
 COPY --from=build-environment /opt/foundry/target/release/cast /usr/local/bin/cast
 COPY --from=build-environment /opt/foundry/target/release/anvil /usr/local/bin/anvil
-ENTRYPOINT ["/bin/sh", "-c"]
+RUN useradd -ms /bin/ash foundry
```

Comment:
if I read the docs correctly, `-m` will create the user's home dir and `-s` use the shell

is there a reason you're using `ash` here instead of `sh`?

Reply:
Yes, `ash` is the default shell for root in your image:

```
docker run ghcr.io/foundry-rs/foundry:latest 'grep ^root /etc/passwd' 2>/dev/null
root:x:0:0:root:/root:/bin/ash
```